content/beginner/190_secrets/create-a-secret.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Create a Secret"
+date: 2019-04-09T00:00:00-03:00
+weight: 10
+draft: false
+---
+
+
+#### Encrypt Your Secret
+
+Create a namespace for this exercise:
+
+```bash
+kubectl create ns secretslab
+```
+Output: 
+{{< output >}}
+namespace/secretslab created
+{{< /output >}}
+
+Create a text file contianing your secret:
+```bash
+echo -n "am i safe?" > ./test-creds
+```
+
+
+Create your secret
+```bash
+kubectl create secret \
+        generic test-creds \
+        --from-file=test-creds=./test-creds \
+        --namespace secretslab
+```
```

Comment:
If I copy and paste this in the Cloud9 terminal, the command fails. It seems it may need reformatting.

Reply:
Are you using the copy icon on the Hugo rendered page to copy and paste? or are you manually copying and pasting from the raw source code?

Reply:
The former - copy icon on the Hugo rendered page.